lightkurve/periodogram.py

```diff
@@ -32,12 +38,11 @@ def from_lightcurve(lc):
         nyquist_frequency = 0.5 * (1./((np.median(lc.time[1:] - lc.time[0:-1])*u.day).to(u.second))).to(u.microhertz).value
         frequencies = np.linspace(1, nyquist_frequency, len(lc.time) // 2) * u.microhertz
         lombscargle = LombScargle((lc.time * u.day).to(u.second), lc.flux * 1e6)
-        uHz_conv = 1./(((1./u.day).to(u.microhertz)))
         powers = lombscargle.power(frequencies, method="fast", normalization="psd")
-        powers *= uHz_conv / len(lc.time)  # Convert to ppm^2/uHz
+        powers = powers / (len(lc.time)** .5) * u.microhertz # Huber et. al 2010
```

Comment:
Can you mention the page or equation number from Huber et al 2010?  And perhaps the URL to the paper? Makes it easier for someone to go and read up about it!

Reply:
done!